interpreter/src/testing.rs

```diff
@@ -1,9 +0,0 @@
-use crate::context::Context;
-use crate::objects::ResolveResult;
-use crate::Program;
-
-/// Tests the provided script and returns the result. An optional context can be provided.
-pub(crate) fn test_script(script: &str, ctx: Option<Context>) -> ResolveResult {
-    let program = Program::compile(script).unwrap();
-    program.execute(&ctx.unwrap_or_default())
-}
```

Comment:
Moved this into `lib.rs` `tests` module so it doesn't live in a separate file.

Difference is `use crate::tests` instead of `use crate::testing` and there's less visual noise.